### PR TITLE
Protocol changes (elephant 3)

### PR DESCRIPTION
**Pull request by dgarcia360**
WIP

### CODE REVIEW

File: spec/openapi3.yaml [hunk 1]
```diff
@@ -1188,41 +1188,36 @@ components:
...
-        - 1
-        - 2
-      description: |
-        Identifier of the mosaic property:
-        * 0 - MosaicFlags
-        * 1 - Divisibility
-        * 2 - Duration
-      example: 0
-    MosaicSupplyChangeDirectionEnum:
+        - 0x00 (none) - No flags present.
```
Comment (rg911) [inline spec/openapi3.yaml]: OpenApi Generator return error regarding wrong enum model format

**Comment by rg911** [comment]
Would also suggest to change `Error` model name to e.g. `ModelError` as `Error` is reserved name in the generator

**Comment by dgarcia360** [comment]
@rg911 @fboucquez The spec should be aligned with the latest changes. Although, some more incompatibilities may appear when integrating the SDKs.

## Known incompatibilities

**catapult-rest** does not serialize properly the following endpoints responses.

* **/block/:id/receipts** : Responses are always empty.
* **/account/:id/multisig** : ``accountAddress``, ``multisigPublicKeys`` and ``cosignatoryPublicKeys`` have wrong encoding.
* **/account/:id**: ``linkedAccountKey`` has wrong encoding.

## Open questions

* Are mosaic restrictions #5 and metadata endpoints #6 planned for next release? 
* Is Uint64 array removal https://github.com/nemtech/catapult-rest/issues/84 planned for next release?

**Comment by rg911** [comment]
> * Are mosaic restrictions #5 and metadata endpoints #6 planned for next release?

Mosaic restriction and metadata are both implemented in TS SDK and WIP in Java. So I'd expect them to be included in next release

> * Is Uint64 array removal [nemtech/catapult-rest#84](https://github.com/nemtech/catapult-

Optional but can go in as well. Rest is almost ready anyway

**Comment by Vektrat** [comment]
since greg assigned steven to the uint64 array removal the past week, i assumed there was some kind of urgency to this, so we have proceeded to implement it, please check the issue update to plan accordingly

https://github.com/nemtech/catapult-rest/issues/84#issuecomment-527834360

**Comment by evias** [comment]
in fact UInt64 format fix was first introduced in SDK PR to permit string format. Then discussion with core devs made us think this formatting can happen on top layer with REST. 

Thanks for addressing so fast to REST team :+1:

**Pull request merged**